tests/test_core.rs

```diff
@@ -234,7 +234,7 @@ fn count_clones() {
     // Check that RepeatN only clones N - 1 times.
 
     use core::cell::Cell;
-    #[derive(PartialEq, Debug)]
+    #[derive(PartialEq, Eq, Debug)]
```

Comment:
Depends. Might `Foo` be explicitly used to test stuff that only relies on `PartialEq` - and not `Eq`?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq